### ISSUE TITLE
Crashing on win10 when idle

### ISSUE THREAD

**Issue by roarny** [spyder-ide/spyder#13758]
### What steps will reproduce the problem?

Start Anaconda navigator, start spyder 4.1.5, wait about half an hour. Spyder will have closed. (Crashing without error message). When started again, it offers to restore unsaved work. Have performed the usual steps of updating and resetting. 

## Versions

* Spyder version: 4.1.5 
* Python version: 3.8.5
* Qt version: 5.9.7
* PyQt5 version: 5.9.2
* Operating System: Windows 10

### Dependencies

```

# Mandatory:
atomicwrites >=1.2.0           :  1.4.0 (OK)
chardet >=2.0.0                :  3.0.4 (OK)
cloudpickle >=0.5.0            :  1.6.0 (OK)
diff_match_patch >=20181111    :  20200713 (OK)
intervaltree                   :  None (OK)
IPython >=4.0                  :  7.18.1 (OK)
jedi =0.17.1                   :  0.17.1 (OK)
nbconvert >=4.0                :  5.6.1 (OK)
numpydoc >=0.6.0               :  1.1.0 (OK)
paramiko >=2.4.0               :  2.7.2 (OK)
parso =0.7.0                   :  0.7.0 (OK)
pexpect >=4.4.0                :  4.8.0 (OK)
pickleshare >=0.4              :  0.7.5 (OK)
psutil >=5.3                   :  5.7.2 (OK)
pygments >=2.0                 :  2.6.1 (OK)
pylint >=1.0                   :  2.6.0 (OK)
pyls >=0.34.0;<1.0.0           :  0.34.1 (OK)
qdarkstyle >=2.8               :  2.8.1 (OK)
qtawesome >=0.5.7              :  0.7.2 (OK)
qtconsole >=4.6.0              :  4.7.6 (OK)
qtpy >=1.5.0                   :  1.9.0 (OK)
rtree >=0.8.3                  :  0.9.4 (OK)
sphinx >=0.6.6                 :  3.2.1 (OK)
spyder_kernels >=1.9.4;<1.10.0 :  1.9.4 (OK)
watchdog                       :  None (OK)
zmq >=17                       :  19.0.1 (OK)

# Optional:
cython >=0.21                  :  0.29.21 (OK)
matplotlib >=2.0.0             :  3.3.1 (OK)
numpy >=1.7                    :  1.19.1 (OK)
pandas >=0.13.1                :  1.1.1 (OK)
scipy >=0.17.0                 :  1.5.0 (OK)
sympy >=0.7.3                  :  1.6.2 (OK)
```

**Comment by andfoy** [comment]
Hi @roarny, could you please provide a complete description of the state of Spyder before it crashes? Also, does your computer enters sleep or hibernation?

**Comment by roarny** [comment]
@andfoy Spyder 4.1.5 64bit has a few files open, but the crash can happen before any script have been run. No spyder plugins installed. The Spyder settings should differ from the factory defaults only in that a different "main interface" has been selected under "appearance". Saw this problem frequently in 4.1.4 too. 
I've seen it close unexpectedly when I'm working in other programs, so the PC was not in sleep mode while it happened.

**Comment by ifariasg** [comment]
I have this exact same problem.
I usually leave spyder open the night before and since around 3 weeks back by the morning spyder has always crashed, no exception, with no error messages or log. Was the case with spyder 4.1.4 and is still happening with 4.1.5, both 64 bits. In my case I'm using the default "AMD ryzen balanced power plan" which has "hybrid sleep" and "turn off hdd" turned on after X minutes.
The crash is totally random, but always occurs when I'm NOT using spyder actively, but if I'm using other software Spyder crashes in the background.

**Comment by ccordoba12** [comment]
This problem should be fixed in our latest version available in Anaconda (5.3.3). Please update by closing Spyder, opening the Anaconda Prompt and running there the commands mentioned in [our documentation](http://docs.spyder-ide.org/current/installation.html#updating-with-anaconda).
